2015-EDS/JumpZoom/README.md

```diff
@@ -26,7 +26,7 @@ Tap the menu icon to open the navigation drawer, and choose different locations
 You can use the default gestures on the map to zoom in or out and pan the map; this will cancel any current programmatic navigation.
 
 ## Licensing
-Copyright 2015 Esri
+Copyright 2016 Esri
```

Comment:
I updated the README to make it a file changed.  Doc needs to be updated to represent 100.0.0 final release and remove references to beta 2